spec/lib/api/redfish-1.0/system-spec.js

```diff
@@ -134,6 +166,9 @@ describe('Redfish Systems Root', function () {
         this.sandbox.stub(mktemp, 'createFile');
         this.sandbox.stub(fs, 'writeFile');
 
+        redfish.getVendorNameById.resolves();
+        redfish.getVendorNameById.rejects();
+        waterline.nodes.findOne.resolves();
```

Comment:
They are not common stubs that will be used for every case. You could stub them only when you need to use them in a specific case.